java/tools/src/test/resources/orc-file-dump.json

```diff
@@ -461,48 +461,48 @@
               "numHashFunctions": 7,
               "bitCount": 9600,
               "popCount": 238,
-              "loadFactor": 0.024791667237877846,
-              "expectedFpp": 5.756256582500896E-12
+              "loadFactor": 0.024791667,
+              "expectedFpp": 5.7562566E-12
```

Comment:
The `com.google.gson.stream.JsonWriter#value(float)` method is not provided in gson 1.9.0, but is provided in gson 1.9.1.

https://github.com/google/gson/pull/2130

This causes the test to fail after upgrading gson, because in version 2.9.0, the generated json float will be implicitly converted to double.

### json format
https://github.com/apache/orc/blob/bd2af54a7aa8865f220d862eab024c27983ab85e/java/tools/src/java/org/apache/orc/tools/JsonFileDump.java#L426-L432

### not json format
https://github.com/apache/orc/blob/bd2af54a7aa8865f220d862eab024c27983ab85e/java/tools/src/java/org/apache/orc/tools/FileDump.java#L719-L726

For other test files in non-json format, we can see that the loadFactor is float.

https://github.com/apache/orc/blob/bd2af54a7aa8865f220d862eab024c27983ab85e/java/tools/src/test/resources/orc-file-dump-bloomfilter2.out#L68-L69